.charts/overwhelm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: overwhelm
-version: 0.0.3
+version: 0.0.4
```

Comment:
This is technically a breaking change, so you should update it to 1.0.0 instead

Reply:
Would it really be breaking if we leave the default behavior the same? Or is it technically breaking because the values.yaml has a new field?